code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -16,6 +17,25 @@
 			return
 	..()
 
+/obj/item/clothing/mask/breathmuzzle
+	name = "surgery mask"
+	desc = "To silence those pesky patients before putting them under."
+	icon_state = "breathmuzzle"
+	inhand_icon_state = "breathmuzzle"
+	clothing_flags = MASKINTERNALS | BLOCKS_SPEECH
+	gas_transfer_coefficient = 0.1
+	permeability_coefficient = 0.01
+	equip_delay_other = 25 // my sprite has 4 straps, a-la a head harness. takes a while to equip, longer than a muzzle
+
+/obj/item/clothing/mask/breathmuzzle/attack_paw(mob/user, list/modifiers)
+	/// The breathmuzzle is similar enough to a regular muzzle that similar rules would apply to both.
+	if(iscarbon(user))
+		var/mob/living/carbon/carbon_user = user
+		if(src == carbon_user.wear_mask)
+			to_chat(user, "<span class='warning'>You need help taking this off!</span>")
```

Comment:
Really, this should be an element as well, but only having it in these two places is fine enough for now.